quickwit-serve/src/node_info_handler.rs

```diff
@@ -55,5 +56,50 @@ fn node_config_handler(
 }
 
 async fn get_config(config: Arc<QuickwitConfig>) -> Result<impl warp::Reply, Rejection> {
-    Ok(warp::reply::json(&*config))
+    // We need to hide sensitive information from metastore URI.
+    let mut config_to_serialize = (*config).clone();
```

Comment:
This is lame... I decided to return the raw quickwit config and hide sensitive data from the metastore uri.
It's interesting to get that info but it does not give you the real config in use (some values if absent are then deduced from another value).

Reply:
I think your comment is obsolete. You now clone the config and edit the value manually (which is nicer)